packages/styling/src/utilities/icons.ts

```diff
@@ -108,11 +119,47 @@ export function registerIcons(iconSubset: IIconSubset, options?: Partial<IIconOp
   }
 }
 
+/**
+ * Unregisters icons by name.
+ *
+ * @param iconNames - List of icons to unregister.
+ */
+export function unregisterIcons(iconNames: string[]): void {
+  const options = _iconSettings.__options;
+
+  for (const iconName of iconNames) {
+    const normalizedIconName = normalizeIconName(iconName);
+    if (_iconSettings[normalizedIconName]) {
+      delete _iconSettings[normalizedIconName];
+    } else {
+      // Warn that we are trying to delete an icon that doesn't exist
+      if (!options.disableWarnings) {
+        warn(`The icon "${iconName}" tried to unregister but was not registered.`);
+      }
+    }
+
+    // Delete any aliases for this iconName
+    if (_iconSettings.__remapped[normalizedIconName]) {
+      delete _iconSettings.__remapped[normalizedIconName];
+    }
+
+    // Delete any items that were an alias for this iconName
+    const remappedKeys = Object.keys(_iconSettings.__remapped).filter((key: string) => {
+      return _iconSettings.__remapped[key] === normalizedIconName;
+    });
```

Comment:
another case where we don't have { return }, we can just do => _iconSettings._....

Reply:
Thanks for pointing these out, that isn't my goto syntax but I'll update these and will be on the lookout in the future